cmd/bootstrap/run/qc.go

```diff
@@ -175,3 +190,34 @@ func GenerateQCParticipantData(allNodes, internalNodes []bootstrap.NodeInfo, dkg
 
 	return qcData, nil
 }
+
+// GenerateQCSignerParticipantData generates participant data for creating QC.
+// Caller needs to pass dkg participants that will be taking part in building QC.
+func GenerateQCSignerParticipantData(internalNodes []bootstrap.NodeInfo, dkgParticipant dkg.DKGParticipantPriv, dkgData inmem.EncodableDKG) (*ParticipantData, error) {
```

Comment:
As mentioned in the other comment, we don't actually need a private key to generate a QC from existing votes. 

I wrote up a function that should do the trick to generate a signer/validator which can be used, without needing to load any private data (which ultimately isn't used anyway):

```go
func createAggregator(participantData *ParticipantData) (hotstuff.Validator, hotstuff.SignerVerifier, error) {

	identities := participantData.Identities()

	var (
		unusedLocalNodeID                   = flow.ZeroID
		unusedModuleLocal module.Local      = nil
		unusedPrivateKey  crypto.PrivateKey = nil
	)

	// create consensus committee's state
	committee, err := committees.NewStaticCommittee(identities, unusedLocalNodeID, participantData.Lookup, participantData.GroupKey)
	if err != nil {
		return nil, nil, err
	}

	merger := signature.NewCombiner(encodable.ConsensusVoteSigLen, encodable.RandomBeaconSigLen)
	stakingSigner := signature.NewAggregationProvider(encoding.ConsensusVoteTag, unusedModuleLocal)
	beaconVerifier := signature.NewThresholdVerifier(encoding.RandomBeaconTag)
	beaconSigner := signature.NewThresholdProvider(encoding.RandomBeaconTag, unusedPrivateKey)
	beaconStore := signature.NewSingleSignerStore(beaconSigner)
	signer := verification.NewCombinedSigner(committee, stakingSigner, beaconVerifier, merger, beaconStore, flow.ZeroID)

	validator := validator.New(committee, new(mocks.ForksReader), signer)

	return validator, signer, nil
}
```